runtime/util_core/vmargs_core.c

```diff
@@ -201,8 +201,14 @@ parseOptionsFileText(J9PortLibrary* portLibrary, const char* fileText, J9JavaVMA
 					 */
 					memcpy(nextOptionPosition, tokenStart, argumentLength);
 					/* The first argument is at the head of the buffer and is the same pointer as the allocated buffer */
-					optArg = newJavaVMArgInfo(vmArgumentsList, nextOptionPosition,
+					if (isDefault) {
```

Comment:
These two arms are the same apart from the third argument, right?  If so, use the ternary operator to generate the third argument.